### ISSUE TITLE
Deprecation warnings from Django 3.1

### ISSUE THREAD

**Issue by ShaheedHaque** [viewflow/viewflow#294]
With Django 3.1, the following messages appear:

```
WARNINGS:
viewflow.Process.data: (fields.W904) django.contrib.postgres.fields.JSONField is deprecated. Support for it (except in historical migrations) will be removed in Django 4.0.
        HINT: Use django.db.models.JSONField instead.
viewflow.Task.data: (fields.W904) django.contrib.postgres.fields.JSONField is deprecated. Support for it (except in historical migrations) will be removed in Django 4.0.
        HINT: Use django.db.models.JSONField instead.
```

**To Reproduce**
1. Upgrade Django to 3.1.2.
2. Run some tests which invoke Django views:
    ```
    py.test -slv test_with_django_view.py
    ```
3. Scroll down to 'warnings summary'
4. See messages

**Versions**
 - Django Viewflow: django_viewflow-1.6.1-py3.8.eg
 - Python 3.8
 - Django 3.1.2

**Expected behavior**
No deprecation warning should be emitted.

**Comment by ShaheedHaque** [comment]
Also saw the following:
```  /usr/local/lib/python3.8/dist-packages/django_viewflow-1.6.1-py3.8.egg/viewflow/flow/viewset.py:75: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
    url('', include(self.get_list_urls()), {'flow_class': self.flow_class}),
  /usr/local/lib/python3.8/dist-packages/django_viewflow-1.6.1-py3.8.egg/viewflow/mixins.py:49: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
    url(r'^(?P<process_pk>\d+)/{}/(?P<task_pk>\d+)/detail/$'.format(self.name),
  /usr/local/lib/python3.8/dist-packages/django_viewflow-1.6.1-py3.8.egg/viewflow/nodes/view.py:252: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
    urls.append(url(r'^(?P<process_pk>\d+)/{}/(?P<task_pk>\d+)/unassign/$'.format(self.name),
```

**Comment by ShaheedHaque** [comment]
The JSONField issues are resolved by viewflow/jsonstore#5.
